frontend/src/views/PublicCanteenStatisticsPage/index.vue

```diff
@@ -557,8 +600,15 @@ export default {
       let newLocationText = this.createLocationText()
       this.loadStatistics(newLocationText, query)
     },
+    updateAllStatistics() {
```

Comment:
Je trouve le nom confusant ayant `updateStatistics` et `updateAllStatistics`... ça laisse supposer que la première fait un update partiel ? 

De plus, cette fonction est utilisé une seule fois, et ajoute une seule ligne (`this.locationText = ""`) On n'a peut être pas besoin de créer une fonction à part, et ça serait plus clair si on utilise cette ligne additionnelle à l'endroit où la fonction est utilisée.